crates/swc/src/config/mod.rs

```diff
@@ -1136,6 +1274,8 @@ pub struct BuiltInput<P: swc_ecma_visit::Fold> {
     /// `pass`.
     pub minify: bool,
     pub external_helpers: bool,
+    /// Path to the source map file.
+    pub source_map_path: Option<PathBuf>,
```

Comment:
I'm not merging this PR because of this

Reply:
I know that this is a large PR and I understand that you are doing this voluntarily. I apologize for the inconvenience.

Reply:
No, I'm talking about adding this field. I meant removing this field by `implicit` path.

https://github.com/swc-project/swc/pull/6817#discussion_r1083376046

Reply:
I can't follow. It would be nice if you could provide me a more detailed example. Thank you.